### PR TITLE
Redirect functional test to use testrepo #131

### LINKED ISSUE #131: Redirect functional test to use testrepo
Functional is performing the test on repositories that are beyond our control.

This means that if the repositories' owners make any changes to their repository, 
it will affect our test results.

Let's use the state-preserved, forked repositories under our organization instead.

### PR DESCRIPTION
**Pull request by yamidark**
Fixes #131 
```
Functional is performing the test on repositories that are beyond our control.

This means that if the repositories' owners make any changes to their repository,
it will affect our test results.

Let's use the state-preserved, forked repositories under our organization instead.
```

### CODE REVIEW

**Comment by yamidark** [comment]
@yong24s @ongspxm @eugenepeh for your review

File: src/functional/resources/sample_full.csv [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Organization,Repository,branch,Student1 Git ID,Student1 Display Name,Student1's Git Alias,Student2 Git ID,Student2 Display Name,Student2's Git Alias,Student3 Git ID,Student3 Display Name,Student3's Git Alias,Student4 Git ID,Student4 Display Name,Student4's Git Alias
-CS2103AUG2017-W09-B1,main,master,nbriannl,,,zacharytang,Zachary Tang,Zachary Tang,April0616,Fan Yuting,LAPTOP-7KFM2KSP\User;Fan Yuting,CindyTsai1,Cindy Tsai,YuHsuan
-CS2103AUG2017-W09-B2,main,master,charlesgoh,,Charles Goh,Esilocke,,,jeffreygohkw,,Jeffrey Goh,wangyiming1019,,ACER\kyle;Wang Yiming
+reposense,testrepo-Beta,master,nbriannl,,,zacharytang,Zachary Tang,Zachary Tang,April0616,Fan Yuting,LAPTOP-7KFM2KSP\User;Fan Yuting,CindyTsai1,Cindy Tsai,YuHsuan
+reposense,testrepo-Charlie,master,charlesgoh,,Charles Goh,Esilocke,,,jeffreygohkw,,Jeffrey Goh,wangyiming1019,,ACER\kyle;Wang Yiming
```
Comment (eugenepeh) [inline src/functional/resources/sample_full.csv]: just a feedback from prof earlier - try not to set their name as display name in the report.

**Comment by eugenepeh** [comment]
> Looks gud, but where my alpha though...

Alpha is for unit tests

**Comment by yamidark** [comment]
Resuming on this PR since #130 has been merged.

**Comment by yamidark** [comment]
@eugenepeh As suggested last time, I have set the display name on the report to be the 1st 3 characters of the author display name/gitID.
@ongspxm @AdityaA1998 @yong24s @eugenepeh for your review again

**Comment by eugenepeh** [comment]
>@eugenepeh As suggested last time, I have set the display name on the report to be the 1st 3 characters of the author display name/gitID.

@yamidark you've mistaken. What I meant was, change the `display name` section of the sample repos in csv config.

**Comment by yamidark** [comment]
>What I meant was, change the display name section of the sample repos in csv config.

Oh, misunderstood that. So the we should change the display name for `sample_full.csv` and `sample_small.csv` at the root directory too?

**Comment by eugenepeh** [comment]
I think we can remove `sample_full.csv`, and make `sample_small.csv` contains 2 repository worth of authors.

`sample_small.csv` can be renamed to `sample.csv` and update the user guide as well.

**Comment by yamidark** [comment]
>Also, can consider taking this chance to move config.csv to the dashboard-demo

Cant move it over from this PR :sweat_smile:, so will probably just delete the `config.csv` here, and make another PR in that repo (with the updated format)?

**Comment by yamidark** [comment]
@eugenepeh for your review again.

**Comment by yong24s** [comment]
LGTM

**Comment by yamidark** [comment]
@damithc Prof, for your review.

**Pull request merged**